Source/common/SNTConfigurator.m

```diff
@@ -563,6 +568,14 @@ - (NSString *)bannedBlockMessage {
   return self.configState[kBannedBlockMessage];
 }
 
+- (NSString *)bannedUSBBlockMessage {
+  return self.configState[kBannedUSBBlockMessage];
+}
+
+- (NSString *)remountUSBBlockMessage {
+  return self.configState[kRemountUSBBlockMessage];
```

Comment:
Same as above - Comment in `SNTConfigurator.h` says a reasonable default is provided if not configured, but no default is provided.